### DISCUSSION TITLE
[Question]: V0.6.10 google AI not accessible，Error fetching from....

### DISCUSSION THREAD

**Discussion by admlke** [danny-avila/LibreChat#2263] [Troubleshooting]
### What is your question?

how to solve such a problem?
`error: [handleAbortError] AI response error; aborting request: [GoogleGenerativeAI Error]: Error fetching from https://generativelanguage.googleapis.com... [truncated]
`

### More Details

`2024-04-01 13:39:04 error: [handleAbortError] AI response error; aborting request: [GoogleGenerativeAI Error]: Error fetching from https://generativelanguage.googleapis.com... [truncated]
2024-04-01 13:39:04 warn: AI Response error for conversation 96fbd633-bffe-47e2-97a1-65772a5f02a8 likely caused by Google censor/filter
`

### What is the main subject of your question?

UI, Endpoints

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
If it happens on every message: Connection issue. Google is region locked, I forget which regions it allows.

**Reply by edge7** [reply]
Am having the same issue. I tried to set the proxy variable but did not work.
Any alternative way? I was thinking of doing something at docker level but it's not straightforward

**Reply by danny-avila** [reply]
PROXY does not work with Google, unfortunately. need a global proxy setup: https://github.com/danny-avila/LibreChat/issues/1372
